addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -2235,12 +2331,11 @@ registry.background = SnippetOptionWidget.extend({
      */
     background: async function (previewMode, widgetValue, params) {
         if (previewMode === 'reset') {
-            // No background has been selected and we want to reset back to the
-            // original custom image
-            await this._setCustomBackground(this.__customImageSrc, previewMode);
-            return;
+            return this._setCustomBackground(this.__customImageSrc, previewMode);
+        }
+        if (!previewMode) {
+            this.__customImageSrc = widgetValue;
         }
-
         if (widgetValue) {
             this.$target.css('background-image', 'url(\'' + widgetValue + '\')');
             this.$target.removeClass('oe_custom_bg').addClass('oe_img_bg');
```

Comment:
Probably wrong to not have adapted this ? I suppose that `oe_custom_bg` can safely be always added now or that we can remove the class and all its uses if possible ?

Reply:
A cursory search through the codebase seems to indicate that it's no longer used for anything: it has no CSS rules and it's only touched by javascript here and in parallax, and in both places it's only to put the class on an element without doing anything with it (supposedly so that something else can affect it, but there is nothing else that does anymore)

anything I should be careful of when removing it?

Reply:
Yes, it is always custom bg now so you can just remove all occurences